examples/objectdetection/android/app/src/main/java/com/objectdetection/MainActivity.kt

```diff
@@ -19,4 +21,8 @@ class MainActivity : ReactActivity() {
    */
   override fun createReactActivityDelegate(): ReactActivityDelegate =
       DefaultReactActivityDelegate(this, mainComponentName, fabricEnabled)
+
```

Comment:
I'm curious why this?

Reply:
> This change is required to avoid crashes related to View state being not persisted consistently across Activity restarts.

https://reactnavigation.org/docs/getting-started#installing-dependencies-into-a-bare-react-native-project -